dice-game-ui-angular/src/app/game/game.component.html

```diff
@@ -1,2 +1,25 @@
-<h1>Game</h1>
+<div *ngIf="gameFinished; else playControls">
+    <img *ngIf="dice1" class="dice-img" src="/assets/images/dices/dice-{{dice1}}.png" alt="{{dice1}}"/>
```

Comment:
In fact, no. I set dices to zeros after every turn. Thus, it's possible to have "empty" in dices. Moreover, I show dices after the game was finished only if the player couldn't place a new rectangle on the field (else he couldn't understand why the game is over) and do not show if the field is full because it's obvious, I guess.

Reply:
Ok, I haven't dived deep into game code yet.